include/common/frame.rs

```diff
@@ -0,0 +1,9 @@
+use crate::include::dav1d::headers::Dav1dFrameHeader;
+
+
+/// Checks whether the frame type is `INTER` or `SWITCH`.
```

Comment:
As we clean up the code now, should we be adding back the comments now?  It seems like a good opportunity for it now.  Should I do that, too?

Reply:
Yeah, please do! I've been adding them in where I find them in the original C.